src/engraving/libmscore/spanner.cpp

```diff
@@ -1069,6 +1077,9 @@ void Spanner::setEndElement(Element* e)
     }
 #endif
     _endElement = e;
+    if (e && _tick > Fraction() && ticks() == Fraction()) {
+        setTicks(e->tick() - _tick);
```

Comment:
only needed for note anchored lines, which never get their `ticks` property updated properly when creating or when loading